test/util.js

```diff
@@ -169,6 +169,18 @@ return {
 	/**
 	 * Etc
 	 */
+	assertRuntimeBind: function( assert, formatter, runtimeKey, generatorString, runtimeArgsFn ) {
+		assert.ok( formatter );
+		assert.equal( typeof formatter, "function" );
+		assert.ok( "runtimeKey" in formatter );
+		assert.equal( formatter.runtimeKey, runtimeKey );
```

Comment:
This assertion seems brittle, in that it'll have to be updated whenever the formatter/parser somehow changes form, independent of the underlying algorithm to generate the key. Does it really makes sense to hardcode the expected key in so many places?